tests/test_benchmark.py

```diff
@@ -252,11 +289,30 @@ def benchmark_dhfr(verbose=False, num_batches=100, steps_per_batch=1000):
     )
 
 
+def prepare_hif2a_initial_states(st, host_system, host_coords, host_box):
+    st = rbfe.SingleTopology(st.mol_a, st.mol_b, st.core, st.ff)
+    host_config = rbfe.HostConfig(host_system, host_coords, host_box)
+    temperature = 300.0
+    lamb = 0.1
+    initial_state = rbfe.setup_initial_states(st, host_config, temperature, [lamb], seed=2022)[0]
```

Comment:
nit: should this name be pluralized?
```suggestion
    initial_states = rbfe.setup_initial_states(st, host_config, temperature, [lamb], seed=2022)[0]
```

Reply:
sorry, I misread. Deleting this comment..